test/avocado/run-tests

```diff
@@ -53,6 +53,30 @@ def prepare_avocado_tests(machine):
     machine.execute(command="mkdir -p " + avocado_results_dir)
     machine.upload([os.path.join(machine.test_dir, "avocado")], machine_test_dir)
 
+def tap_output(machine):
+    json_info = json.loads(machine.execute(command="cat " + os.path.join(avocado_results_dir, "latest/results.json"), quiet=True))
+    print "\nTAP output -------------------------"
+    print "1..%s" % len(json_info['tests'])
+    counter = 1
+    for t in json_info['tests']:
+        test_status = t['status']
+        test_name = t['test']
+        test_log = t['logfile']
+        test_time = t['time']
+        print "# ----------------------------------------------------------------------"
+        print "# %s (time %d secs)" % (test_name, test_time)
+        print "#"
+        if test_status == 'PASS':
+            print "ok %s %s (time %d secs)" % (counter, test_name, test_time)
+        else:
+            print "not ok %s %s (time %d secs)" % (counter, test_name, test_time)
+        print machine.execute(command="cat " + test_log, quiet=True)
```

Comment:
I can move it before result output.
I was not sure when I've read TAP specification and I understood it that it should be after. I'll move it before.